src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault/Generated/Models/SasDefinitionAttributes.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator 1.0.0.0
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+namespace Microsoft.Azure.KeyVault.Models
+{
+    using Azure;
+    using KeyVault;
+    using Rest;
+    using Rest.Serialization;
+    using Newtonsoft.Json;
+    using System.Linq;
+
+    /// <summary>
+    /// The sas definition management attributes.
+    /// </summary>
+    public partial class SasDefinitionAttributes
+    {
+        /// <summary>
+        /// Initializes a new instance of the SasDefinitionAttributes class.
+        /// </summary>
+        public SasDefinitionAttributes() { }
+
+        /// <summary>
+        /// Initializes a new instance of the SasDefinitionAttributes class.
+        /// </summary>
+        /// <param name="enabled">Determines whether the object is
+        /// enabled.</param>
+        /// <param name="created">Creation time in UTC.</param>
+        /// <param name="updated">Last updated time in UTC.</param>
+        public SasDefinitionAttributes(bool? enabled = default(bool?), System.DateTime? created = default(System.DateTime?), System.DateTime? updated = default(System.DateTime?))
+        {
+            Enabled = enabled;
+            Created = created;
+            Updated = updated;
+        }
+
+        /// <summary>
+        /// Gets or sets determines whether the object is enabled.
```

Comment:
This is all generated code, The "gets or sets" was prefixed by generator, I'll update the swagger with description of field as "the enabled state of the object" so that the generated text becomes what you want